lib/models/movie.js

```diff
@@ -11,5 +11,21 @@ module.exports = Bookshelf.Model.extend({
       release_year: this.get('release_year'),
       object: 'movie'
     };
+  },
+  filter: function(filter) {
+    return this.query(function (qb) {
+      if (filter.release_year) {
+        qb.where('release_year', filter.release_year);
+      }
+      if (filter.release_year_start && filter.release_year_end) {
```

Comment:
You probably want these two conditions in two different clauses such that you can search for movies starting or ending in a particular year rather than in a set interval of time.

For example, if I go to `GET /movies?release_year_start=2000` I'd expect to get all movie that started in the year 2000 or later. This is currently not possible as `filter.release_year_end = undefined`.